Library/Homebrew/cmd/fetch.rb

```diff
@@ -1,3 +1,23 @@
+#:  * `fetch` [`--force`] [`-v`] [`--devel`|`--HEAD`] [`--deps`] [`--build-from-source`|`--force-bottle`] <formulae>:
+#:    Download the source packages for the given <formulae>.
+#:    For tarballs, also print SHA-1 and SHA-256 checksums.
```

Comment:
Should be fixed there as well, really. I can do it but don't know if you'd prefer to just wrap that change into this PR?